aten/src/ATen/native/group_norm.cpp

```diff
@@ -119,7 +119,12 @@ std::tuple<Tensor, Tensor, Tensor> native_group_norm_backward(
   c10::MaybeOwned<Tensor> gamma_maybe_owned =
       at::borrow_from_optional_tensor(gamma_opt);
   const Tensor& gamma = *gamma_maybe_owned;
-
+  TORCH_CHECK(
```

Comment:
Hey @CaoE, could I get some context about the assertion regarding the `memory_format` that was added here? Asking because it seems to be causing some bugs

Reply:
Hey @mikaylagawarecki, the assertion here is to make sure that memory_format of X is the same as dY, otherwise backward kernel will generate wrong results.  If we meet the case where CL propagation it breaked, I think we may need to remove this assertion and convert the memory_format of dY to that of X.